WcaOnRails/app/views/delegate_reports/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= simple_form_for(@delegate_report) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.input :content %>
+  </div>
+
+  <div class="form-actions">
+    <%= f.button :submit %>
+  </div>
+<% end %>
```

Comment:
Hmm, this file seems to be unused.
Btw. content doesn't have markdown editor set up.